### DISCUSSION TITLE
windows ethernet drive can not be installed

### DISCUSSION THREAD

**Discussion by scottjl** [eduardofilo/RG350_adam_image#110] [General]
tried to install the ethernet driver referenced under "System access" Windows 10 will not install this driver, is an updated driver available?

https://github.com/SeongGino/RetroGame350-AppRepo/blob/master/RG350-signed_driver.zip

thank you!

**Comment by eduardofilo** [comment]
I don't know what to say. Although I usually use Linux, I installed that driver several months ago in a Windows partition and it works since then. I don't remember problems installing it.

What problem it gives to you?
What should happen after installing it is that this device appears in Network adapters:

![rndis](https://user-images.githubusercontent.com/2923728/140618504-f5620543-8f0e-432d-a788-60d5d8be2983.png)

**Comment by scottjl** [comment]
Using Windows 10, I plug in via USB, it sees a device, I tell it to load the driver, point it at the unzipped folder. It simply doesn't see those files as a valid driver. Nothing loads. I can use it in MTP mode just fine and access the file system, just wasn't what I was looking to do.

**Comment by eduardofilo** [comment]
I have uninstalled my driver to test the installation with the zip file with drivers indicated in README and I have been able to do it without problems. I have recorded the session. Check it out and tell me if you have been able to follow those steps. Naturally the console has been turned on and connected by USB to the computer during the whole process.

https://www.youtube.com/watch?v=Ib5qfYv6ijc

**Comment by scottjl** [comment]
i have tried. but windows just won't load those drivers for me. i am able to access the filesystem by mass storage mode, so that's good enough for me for now. thanks.

**Comment by eduardofilo** [comment]
In that case it must be a specific problem on your machine. I'm going to close the issue because in addition to the fact that I can't do much more than sit in front of your machine to locate the problem, I don't think it is an incident in the image but in your machine. Sorry.

Make use of the MTP mode or try also testing the `Ethernet > ECM` mode in the `USB Mode` application to see if you have more luck.
